### PR TITLE
Feature/gm finalize

### PR DESCRIPTION
**Pull request by kirilknysh**
Add styling to GM view:
image

### CODE REVIEW

File: public/game-master/views/app-view.coffee [hunk 1]
```diff
@@ -66,11 +68,16 @@ AppView = (context) ->
...
 					nxt.Element 'div',
 						nxt.Class 'controls-selector-banned-container'
-						nxt.Text 'banned chars'
-
-					nxt.Element 'div',
-						nxt.Class 'master-controls-puzzle'
-						nxt.Text 'HTML GOES HERE'
+						nxt.Text 'Banned chars'
+						nxt.Binding context.round_phase, (phase) ->
+							#should we each time re-create MatchRenderer? or use CSS to show/hide?
```
Comment (DaQuirm) [inline public/game-master/views/app-view.coffee]: Now that's a good question

File: public/game-master/views/app-view.coffee [hunk 2]
```diff
@@ -66,11 +68,16 @@ AppView = (context) ->
...
+							if phase is RoundPhase.FINISHED or phase is RoundPhase.IN_PROGRESS
+								OccurrenceIndicatorView context.occurrenceIndicator
+
+					nxt.Binding context.round_phase, (phase) ->
+						#should we each time re-create MatchRenderer? or use CSS to show/hide?
```
Comment (DaQuirm) [inline public/game-master/views/app-view.coffee]: Re-creating is alright for now... But let's get back to it later

**Comment by DaQuirm** [comment]
LGTM, but conflicts

File: public/game-master/views/app-view.coffee (hunk 1 above)
Comment (kirilknysh) [inline public/game-master/views/app-view.coffee]: For now - it's much easier to re-render the whole component. Would be nice to have a kind of pre-defined mechanism in `nexus` for such cases.

**Comment by DaQuirm** [comment]
The markup looks sort of weird with white background now. Probably we'll be able to do a dark theme for MatchRenderer too

**Pull request merged**